integration/localnet/bootstrap.go

```diff
@@ -257,10 +265,20 @@ func prepareServices(containers []testnet.ContainerConfig) Services {
 	for _, container := range containers {
 		switch container.Role {
 		case flow.RoleConsensus:
-			services[container.ContainerName] = prepareConsensusService(container, numConsensus, "localnet_access_1_1:9000")
+			services[container.ContainerName] = prepareConsensusService(
+				container,
+				numConsensus,
+				"access_1:9001",
```

Comment:
This address is essentially a constant, I think it would be better to store it with the other constants and use the value directly in `preparareConsensusService`.
https://github.com/onflow/flow-go/blob/1ccefc012fd7bee874f9c094db9dab931463563a/integration/localnet/bootstrap.go#L38-L40